app/src/features/Columns/ColumnSelect.tsx

```diff
@@ -62,13 +67,21 @@ const ColumnSelects = ({
     }
   );
 
-  const { table, column, owner: ownerId } = pendingColumn;
-  const selectedOwner = credentialOwners?.find(({ id }) => id === ownerId);
+  const mapping = useAppSelector((state) =>
```

Comment:
ok I think what you did before was better using `useApiResourcesRetrieveQuery`. Unlike what Jason said, it won't make an API call since it should be already be cached and it's way simpler that way. No need to put the mapping in a slice.

Reply:
See previous comment ?